pkgs/games/simutrans/default.nix

```diff
@@ -11,28 +11,28 @@ let
       else map (name: pakSpec.${name}) (splitString " " paksets)
   );
 
-  ver1 = "120";
-  ver2 = "4";
-  ver3 = "1";
-  version =   "${ver1}.${ver2}.${ver3}";
-  ver_dash =  "${ver1}-${ver2}-${ver3}";
+  ver1 = "121";
+  ver2 = "0";
+  ver3 = "";
+  version  = "${ver1}.${ver2}" + (if ver3 == "" then "" else ".${ver3}");
+  ver_dash = "${ver1}-${ver2}" + (if ver3 == "" then "" else "-${ver3}");
```

Comment:
```suggestion
  ver_dash = "${ver1}-${ver2}${lib.optionalString (ver3 != "") "-${ver3}"}";
```

Reply:
ha, went looking for that and failed. Ty.